mix.exs

```diff
@@ -42,7 +43,14 @@ defmodule SingForNeeds.Mixfile do
       {:plug, "~> 1.7"},
       {:ecto_sql, "~> 3.0"},
       {:phoenix_ecto, "~> 4.0"},
-      {:credo, "~> 1.0.0", only: [:dev, :test], runtime: false}
+      {:credo, "~> 1.0.0", only: [:dev, :test], runtime: false},
+      {:postgrex, ">= 0.0.0"},
+    ]
+  end
+
+  defp aliases do
+    [
+      test: ["ecto.create --quiet", "ecto.migrate", "test"]
```

Comment:
Am I correct that you are effectively overwriting mix test with this? If so, I really don't love this, and without a linked reference it would be hard for me to believe that this is the Phoenix-y way to do this. In the test_helper seems like potentially a better place.

Thinking about it further though, this just doesn't seem like a good idea. It seems to me no task should run migrations without the dev's explicit say-so.